src/modules/core/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -9,7 +8,7 @@ interface Props {
    * array of elements outside of this components and just an array
    * of strings gets passed in. The last active element gets highlighted.
    */
-  elements: MessageDescriptor[];
+  elements: string[];
```

Comment:
I think it'd be wise to allow a mixed array of `strings` **or** `MessageDescriptor`s. We can then use `injectIntl` here and perform the `formatMessage` conditionally on each item.

Reply:
I agree with the suggestions for the breadcrumb component, but I would prefer to do them in the other PR.

Reply:
Sure, makes sense!

> I am not sure why these changes even show up here.

Me neither 🤷‍♂🙂